### ISSUE TITLE
Installation roadmap - 404

### ISSUE THREAD

**Issue by denish-c-vachhani** [magento/merchdocs#1699]
# Bug report

## Description

_What is the issue/current behavior?_

https://docs.magento.com/user-guide/magento/installation.html
Open above url **click on "Installation roadmap"** it will redirect to "https://devdocs.magento.com/guides/v2.4/install-gde/install-roadmap_part1.html?_ga=2.113227276.963052517.1644381423-691481920.1644381423"

### Steps to reproduce

_(OPTIONAL) What needs to be done to replicate this issue? Provide Gist if needed._

1. First Step - Open https://docs.magento.com/user-guide/magento/installation.html
1. Second Step - Click on **"Installation roadmap"**
1. Third step - link goes **Page not found - 404**

## Expected result

_When this issue is resolved, what is the expected result/behavior?_

**"Installation roadmap"** should be properly interlink and redirect should not be 404 page.

## Possible solutions

_(OPTIONAL) What would a solution for this issue look like?_

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Additional information

_(OPTIONAL) What other information can you provide about this issue?_

**Comment by dobooth** [comment]
Yes, let's link it to https://devdocs.magento.com/guides/v2.4/install-gde/install-flow-diagram.html
